openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSettingLeftPanel.tsx

```diff
@@ -34,7 +38,9 @@ const GlobalSettingLeftPanel = () => {
   return (
     <Menu
       className="global-setting-left-panel"
-      defaultSelectedKeys={[tab]}
+      defaultSelectedKeys={[
+        isUndefined(tab) ? GlobalSettingOptions.USERS : tab,
+      ]}
```

Comment:
Can we make sure this via router redirect ?

Reply:
yes, but for here if a user lands on `/settings` route at that time we don't have `:tab` param there so that's why giving default value here,  and in routes, we will redirect to 'users' route
let me know if this can be improved, open to suggestions